src/main/java/com/videoclub/suriken/api/RenterController.java

```diff
@@ -14,7 +14,6 @@
 
     private final RenterService renterService;
 
-    @Autowired
```

Comment:
Obavezno da se zna zasto ne mora autowired ovde, tj sta radi autowired i koja je razlika izmedju Autowired i konstruktora za dep injection. Vec sam to napisao, ali jos jednom